### ISSUE TITLE
Cannot start the Desktop Clock App V4.0 or install it with the msi file in Windows 11

### ISSUE THREAD

**Issue by MAP6768** [danielchalmers/DesktopClock#39]
For some reason, while I can install previous versions on my computer (Windows 11), when i try to start the DesktopClock.exe V.4
or the install.desktopClock.msi nothing happens.
I just cannot install or use the latest version while I can do it with previous versions.    
I love your App. Your assistance would be appreciated.
PS I even disabled Norton 360.

**Comment by danielchalmers** [comment]
Hi, were you using the countdown mode?

This should be fixed in [v4.1.0](https://github.com/danielchalmers/DesktopClock/releases/tag/v4.1.0) :)

**Comment by MAP6768** [comment]
Thanks Daniel.

I never use the countdown feature.

 

However, while I could not use or even download completely the file DesktopClock.exe (even when indicating that it was safe), I was able to successfully download the .msi file with Edge browser (not Chrome) after insisting that I keep it.

Something in those files triggers the threat protect mecanism.

Be that as it may, I was finally able to install DesktopClock with the .msi file.

However, the "Show background" feature does not work. The background is always there whether I toggle or not the Show Background. 

In the settings file, changing the "BackgroundEnabled": from true to false, does not change anything.

I had to put the "BackgroundOpacity": to 0.0,  

 

Cheers,

 

Michel
